edb/edgeql/parser/grammar/ddl.py

```diff
@@ -851,7 +851,12 @@ def reduce_create_data_branch(self, *kids):
 #
 class DropBranchStmt(Nonterm):
     def reduce_DROP_BRANCH_DatabaseName(self, *kids):
-        self.val = qlast.DropDatabase(name=kids[2].val)
+        _, _, name = kids
+        self.val = qlast.DropDatabase(name=name.val)
+
+    def reduce_DROP_BRANCH_RESET_CONNECTIONS_DatabaseName(self, *kids):
```

Comment:
Not sure I like the formulation. It doesn't read very well and would scale poorly if we add more options. I actually like PostgreSQL's `drop database foo [with option, ...]` somewhat more. I also like `force` as a shorter spelling (a-la Postgres also)

Reply:
Sure